docs/articles/documentation/extending-testcafe/custom-browser-provider-plugin/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: docs
+title: Custom Browser Provider Plugin
+permalink: /documentation/extending-testcafe/custom-browser-provider-plugin/
+---
+# Custom Browser Provider Plugin
+
+To work with different browsers during testing, TestCafe can use the [built-in library](../../using-testcafe/common-concepts/using-browsers.md#built-in-library) or [browser provider plugins](../../using-testcafe/common-concepts/using-browsers.md#browser-provider-plugins).
+If you need to use a specific browser and perform actions on the browser window in a custom way, you can create a **custom browser provider plugin**. For this purpose, use the [TestCafe browser provider generator](https://github.com/DevExpress/generator-testcafe-browser-provider).
```

Comment:
I don't like the whole paragraph to be honest, because it doesn't describes what browser provider plugin is. Browser provider is a plugin that tells TestCafe how to start/gracefully close and perform actions with browser windows from specific sources (e.g. cloud services)